src/components/OrganizationsList/index.jsx

```diff
@@ -1,159 +1,171 @@
 import React, {Component} from 'react';
-import { Table, Button, InputGroup, InputGroupAddon, Input } from 'reactstrap';
 import Api from '../../utils/api';
-import Item from './Item';
 import AlertMessage from '../../sharedComponents/AlertMessage';
-import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import './OrganizationsList.css'
 
 export default class DriversList extends Component {
 
-  state = {
-    organizations: [],
-    errors: null,
-    flash: null,
-    selectedPage: 1,
-    name: ''
+  constructor() {
+    super()
+    this.state = {
```

Comment:
I might be outdated but as far as I remember, setting the state with class properties is kinda a letdown for the performance, because it needs to be transpiled first into something that react can use, or also can lead to the constructor of the component being called without setting the states. It's safer to use the constructor and then filling the states inside it, because : 
1.- It's something that doesn't need to be translated.
2.- The constructor sets the states first and then builds the component. 
I have been doing that change in every component I have worked on.

Reply:
We are using create-react-app, you can safely use the class property 'way', babel will transpile the code to use the constructor 'way'

Reply:
The thing here is that I already changed almost all the class properties into constructor statements, I wasn't warned of this and I changed them into what are you seeing today. Changing them back to their original form is a step back (Time that I could be using to fix other stuff on which I'm working on) And yes, babel does transpile the class properties into constructor properties, and after that, the component is build, so, the final product is kinda the same, just with another step added on the top of it. I think this step added while using the class properties is useless.